programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(numbers) {
+    let numbersum = [];
+    for(let a = 0; a < numbers.length; a++) {
+        for(let b = 0; b < numbers.length; b++) {
+            if(a != b) {
+                numbersum.push(numbers[a] + numbers[b]);
+            }
+        }
+    }
+    numbersum.sort(function(a,b){return a-b});
```

Comment:
1. 공백 컨밴션 유의
2. 펑션보다는 arrow펑션 사용
```suggestion
    numbersum.sort((a, b) => a - b );
```